apps/emqx_durable_storage/src/emqx_ds_replication_layer_meta.erl

```diff
@@ -21,20 +21,26 @@
 %% implementation details from this module.
 -module(emqx_ds_replication_layer_meta).
 
+-compile(inline).
```

Comment:
Q: is this an intended change?  Maybe it was forgotten here?

Reply:
Yes, I don't want to write too much table traversal, but also do not like introducing costs of the anonymous function in the general traversal